cloudbuild.yaml

```
@@ -80,0 +1,79 @@
+  # Step 1: Install dependencies
+  - name: 'gcr.io/cloud-builders/npm'
+    args: ['install']
+
+  # Step 2: Conditionally run builds based on _BUILD_ENV
+  - name: 'gcr.io/cloud-builders/npm'
+    entrypoint: 'bash'
+    args:
+      - '-c'
+      - |
+        echo "Build env:" ${_BUILD_ENV}
+        if [ "${_BUILD_ENV}" = "test" ]; then 
```

yamllint

[error] trailing spaces

(trailing-spaces)